pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -654,6 +655,16 @@ func (r *Builder) mapDisks(vm *model.VM, persistentVolumeClaims []*core.Persiste
 			pvcMap[pvc.Annotations[AnnImportBackingFile]] = pvc
 		}
 	}
+
+	var bootDisk int
```

Comment:
@bkhizgiy I don't remember - did you check whether virt-v2v sets the boot order accordingly in the generated VM configuration. if it does, it would be great to take it from there

Reply:
yes I looked, the order in virt-v2v is not sets the boot order, its just presents the disks in there lexicographical order, so we have to to the match ourselves.